src/Faker/Provider/ru_RU/Company.php

```diff
@@ -96,7 +96,7 @@ public static function companyNameSuffix()
         return static::randomElement(static::$companyNameSuffixes);
     }
 
-    public static function inn($area_code = "")
+    public static function inn($area_code = "", $mask = '#######')
```

Comment:
Use mask as parameter is not good idea. I think better is use type of INN: for Person (12 digits) or Company (10 digits)

And then use type inside method for generate mask.

Reply:
Or move INN generator to common function in \Faker\Calculator\Inn and add inn to Person.php too

Reply:
> Or move INN generator to common function in \Faker\Calculator\Inn and add inn to Person.php too

Thank you for your comments. Using Inn base class and divide the INN for the company and user will be more object-oriented way.